prepline_general/api/models/form_params.py

```diff
@@ -100,6 +101,15 @@ def as_form(
             ),
             BeforeValidator(SmartValueParser[bool]().value_or_first_element),
         ] = False,
+        content_type: Annotated[
```

Comment:
What do you think about calling this parameter `content_type_hint` to be more explicit?